index.html

```diff
@@ -150,17 +155,130 @@ <h1 class="greet">
             </ul>
           </div>
           <div class="see-project">
-            <button class="button">See Project</button>
+            <button type="button" class="button">See Project</button>
           </div>
       </div>
       </section>
-      <div class="aboutme_container">
-        <p class="about_me" #id="about_me">About<br> Myself</p>
+      <!----------------------------Desktop--------------------------------->
+      <div class="desktop-work">
+        <div class="desktop_projects" id="desktop_1">
+          <div class="work-img">
+            <img src="assets/Snapshoot Portfoliodesktop.png">
+          </div>
+          <div class="desktop_info">
+            <div class="project-title">Tonic</div>
+            <div class="some-list">
+              <img src="assets/Frame 2.png">
+            </div>
+            <div class="work-info">
+              A daily selection of private and personalised
+              reads ; no accounts
+              or sign-ups required.
+            </div>
+            <div class="langs">
+              <ul class="lang-list">
+                <li class="Html">html</li>
+                <li class="ruby">Ruby on rails</li>
+                <li class="Css">css</li>
+                <li class="js">javascript</li>
+              </ul>
+            </div>
+            <div class="see-project">
+              <button type="button" class="button">See Project</button>
+            </div>
+          </div>
+        </div>
```

Comment:
- [ ] Excellent job fixing the spacing for cards 👍 However, please make sure that your text/heading is not affected with the change. While you have done a great job, but your text/heading is now disturbed. No worries, let's fix this now 💪
- [ ] Also, please make sure that the space between you technology list is according to figma design.
- [OPTIONAL] Isn't it better to have a straight line for your right side of the column (marked in blue). Right now the technologies are slightly towards left whereas everything else is slightly shifted towards right. What do you think can we fix this? 🎯 I know you can do that 💪
## **_Your design previously 👇_**
![image](https://user-images.githubusercontent.com/106119105/213880474-52d47566-057e-43c0-bdd9-2bfbfa860842.png)

## **_Your design now 👇_**
![image](https://user-images.githubusercontent.com/106119105/213880664-57bafced-b400-4084-84b3-b31069bdc081.png)

## **_Figma design 👇_**
![image](https://user-images.githubusercontent.com/106119105/213880543-ab42b0b6-c317-42bb-93e7-078c3f6f1c07.png)

**_Please make sure that this applies to your all project cards_**